e2e/pages/events/manageDocumentsEvent.page.js

```diff
@@ -85,4 +117,19 @@ module.exports = {
     this.uploadDocument(supportingEvidenceDocument.document, index);
     this.selectConfidential(index);
   },
+
+  async setRolePreferences(rolePreferences) {
```

Comment:
how this is related to your story? does not seems to be used

Reply:
mistake, removed 👍